<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>audioextDialog</class>
 <widget class="QDialog" name="audioextDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>113</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>503</width>
    <height>113</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>503</width>
    <height>113</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Audio Extractor</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>491</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Select File</string>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Video File...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pex">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Extract</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Status:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelst">
      <property name="text">
       <string>Ready</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>272</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>16</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>&amp;Cancel</string>
      </property>
      <property name="icon">
       <iconset resource="ExMplayer.qrc">
        <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButtonof">
      <property name="text">
       <string>&amp;Open output</string>
      </property>
      <property name="icon">
       <iconset resource="ExMplayer.qrc">
        <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="ExMplayer.qrc"/>
 </resources>
 <connections/>
</ui>
